src/Symfony/Component/Form/FormEvents.php

```diff
@@ -11,6 +11,12 @@
 namespace Symfony\Component\Form;
 
 /**
+ * To learn more about how form events work check the documentation
+ * entry at {@link http://symfony.com/doc/current/components/form/form_events.html}
```

Comment:
I agree that forcing the user to choose explicitly is a bad UX, especially given that many pages are exactly the same in all maintained versions (because of the way the doc team manages the doc, where all improvements are always done in the older maintained versions being concerned)